luatest/coverage_utils.lua

```diff
@@ -42,6 +53,10 @@ function export.enable()
         for _, item in pairs(export.DEFAULT_EXCLUDE) do
             table.insert(config.exclude, item)
         end
+
+        runner.data = {}
+        find_luas(runner.data, root)
```

Comment:
Accessing undocumented fields of `luacov.runner`. looks dangerous. At least, this requires a comment.

The only alternative I see is writing a coverage report file with zero counters before initializing `luacov.runner`. This could be done with `luacov.stats.save`.

@Totktonada  WDYT?